pkg/slices.go

```diff
@@ -0,0 +1,11 @@
+package pkg
+
+type SliceFlags []string
+
+func (i *SliceFlags) String() string {
```

Comment:
Why adding a func that implements  `fmt.Stringer` to return an empty string.

Maybe it's OK because you don't want this struct to be printed, so maybe a comment here would be meaningful